### PR TITLE
Update nodejs to 6.10.0

### PR DESCRIPTION
**Pull request by nullobject**
This PR updates our nodejs version to LTC 6.10.0.

I also made a small cleanup for the sharejs deps, they seemed to be split across two files for no apparent reason.

### As root
- `cd ~/.babushka/sources/conversation && git pull`
- `initctl stop sharejs.theconversation.edu.au_sharejs`
- `babushka "conversation:nodejs.bin"`
- `initctl start sharejs.theconversation.edu.au_sharejs`

### As the app user
- `rm -rf node_modules && npm i`

### CODE REVIEW

**Comment by nullobject** [comment]
[Upgrade nodejs on production to 6.x](https://trello.com/c/LPwIN6W1/6350-upgrade-nodejs-on-production-to-6-x)

File: packages.rb [hunk 1]
```diff
@@ -25,19 +25,21 @@
...
+    )
   }
-  provides {
-    via :apt, "nodejs ~> #{owner.version}"
-    otherwise "node ~> #{owner.version}"
+  installs {
+    via :apt, "nodejs"
+    via :brew, "node"
   }
+  provides "node ~> #{version}"
```
Comment (yob) [inline packages.rb]: will this eventually fail if `version` is fixed to `6.10.0` and nodesource provides a higher `6.x` release?

Comment (nullobject) [inline packages.rb]: I think it'll pass for any 6.x version number, but 7.x would fail.

Comment (yob) [inline packages.rb]: 👍

**Pull request merged**